### PR TITLE
Document a metrics guide for platforms

### PR DESCRIPTION
**Pull request by kathy-t**
**Description**
This PR modifies the metrics documentation so that it describes the steps that a platform has to take to start submitting metrics to Dockstore. The existing sections are re-arranged a bit which makes it more obvious that the metrics documentation is platform-focused.

I recommend checking this out locally to see the formatting.

**Issue**
[SEAB-5941](https://ucsc-cgl.atlassian.net/browse/SEAB-5941)

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If the PR has any new pages, only **AFTER** the PR is approved by all reviewers, generate a new discourse topic, by running the action at https://github.com/dockstore/dockstore-documentation/actions/workflows/add-discourse-topic.yml. Otherwise we may end up with orphaned topics, which would need manual management.

[SEAB-5941]: https://ucsc-cgl.atlassian.net/browse/SEAB-5941?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/advanced-topics/metrics.rst [hunk 1]
```diff
@@ -12,38 +16,60 @@ Users are able to execute workflows on various platforms using Dockstore's Launc
...
-Select a workflow and click on the Versions tab. Versions that have metrics have a check mark in the Metrics column.
+Getting Started With Submitting Metrics to Dockstore
+----------------------------------------------------
 
-.. figure:: /assets/images/docs/metrics/versions-metrics-column.png
-    :alt: Metrics column in Versions table
+The following sections go over how a platform can submit workflow metrics to Dockstore. 
 
-Select a version with metrics then click on the Metrics tab to view the metrics available.
+.. contents::
```
Comment (kathy-t) [inline docs/advanced-topics/metrics.rst:33]: This is one of the things that you will want to check out locally. This renders to a list of the sub section titles, which are the various steps to submitting metrics

**Review by svonworl — APPROVED** [review]

File: docs/advanced-topics/metrics.rst [hunk 2]
```diff
@@ -12,38 +16,62 @@ Users are able to execute workflows on various platforms using Dockstore's Launc
...
 
-.. figure:: /assets/images/docs/metrics/metrics-tab.png
-    :alt: Metrics tab
+.. contents::
+   :local:
+   :depth: 1
 
-Why would I, a platform owner, want to submit workflow metrics?
----------------------------------------------------------------
+Dockstore will occasionally aggregate the workflow metrics you submit and display them in the :ref:`Dockstore UI<How to view metrics>`.
```
Comment (denis-yuen) [inline docs/advanced-topics/metrics.rst:39]: `occasionally` sounds a bit weird, while we work out out schedule, just drop `occasionally`

File: docs/advanced-topics/metrics.rst (hunk 1 above)
Comment (denis-yuen) [reply docs/advanced-topics/metrics.rst:35]: Built at https://docs.dockstore.org/en/feature-seab-5943-metrics-use-cases/advanced-topics/metrics.html#why-would-i-a-platform-owner-want-to-submit-workflow-metrics

File: docs/advanced-topics/metrics.rst [hunk 3]
```diff
@@ -184,12 +212,41 @@ The curl command looks something like:
...
+Viewing And Updating Submitted Workflow Metrics
+-----------------------------------------------
+
+As a platform owner who previously :ref:`submitted execution metrics to Dockstore<Getting started with submitting metrics>`, you may want to view or update submitted executions.
 
 .. _View submitted execution:
 
-How do I view a submitted execution?
-------------------------------------
+Viewing a submitted execution
```
Comment (denis-yuen) [inline docs/advanced-topics/metrics.rst:248]: Took out "I" here but  added it earlier with "Why Would I, a Platform Owner, Want to Submit Workflow Metrics?"
Just curious

File: docs/advanced-topics/metrics.rst [hunk 4]
```diff
@@ -12,38 +16,62 @@ Users are able to execute workflows on various platforms using Dockstore's Launc
...
-Why would I, a platform owner, want to submit workflow metrics?
----------------------------------------------------------------
+Dockstore will occasionally aggregate the workflow metrics you submit and display them in the :ref:`Dockstore UI<How to view metrics>`.
 
-As a platform owner, workflow metrics indicate to others that your platform is compatible with many workflows on Dockstore. Workflow metrics provide valuable information to users, including information about the resources and time needed to run the workflow. It helps the user determine if the workflow is high quality and likely to function correctly.
+Record workflow execution metrics on your platform
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to collect metrics for executions of public Dockstore workflows that occur on your platform. The mandatory metrics that Dockstore requires are the date of execution and the execution status. 
+If they are available, you may also collect additional metrics such as how long it took the workflow to run, CPU requirements, memory requirements, etc.
```
Comment (denis-yuen) [inline docs/advanced-topics/metrics.rst:45]: `may also` -> `encouraged` maybe

File: docs/advanced-topics/metrics.rst [hunk 5]
```diff
@@ -12,38 +16,62 @@ Users are able to execute workflows on various platforms using Dockstore's Launc
...
+Register a platform partner user on Dockstore
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-How do I submit workflow metrics?
----------------------------------
+Only Dockstore admins, curators, and platform partners can :ref:`submit metrics<Submit executions>` for workflow executions, and :ref:`retrieve and update submitted executions<Viewing And Updating Submitted Workflow Metrics>`.
+If you're a platform owner and you don't have a platform partner user on Dockstore, please:
+
+1. :doc:`Create a Dockstore account </getting-started/register-on-dockstore>`, if you haven't already. You will use the credentials of this user to submit Dockstore metrics.
+2. Contact us via our `GitHub <https://github.com/dockstore/dockstore/issues>`_ issues or open a helpdesk ticket on `Discourse <https://discuss.dockstore.org/>`_ and we will give your Dockstore user a platform partner role.
```
Comment (coverbeck) [inline docs/advanced-topics/metrics.rst:67]: I'd suggest only a helpdesk ticket, and not mention a GitHub issue -- the paranoid side of me doesn't want to potentially identify public elevated privilege users.

**Review by coverbeck — APPROVED** [review]

File: docs/advanced-topics/metrics.rst (hunk 3 above)
Comment (kathy-t) [reply docs/advanced-topics/metrics.rst:248]: The diff is misleading, "Why Would I, a Platform Owner, Want to Submit Workflow Metrics?" wasn't changed (it always had "I")

It doesn't match with the other titles though, so I will remove "I"

**Review by denis-yuen — APPROVED** [review]

**Comment by kathy-t** [comment]
Changed base branch to the 1.15.2 hotfix branch because the ticket milestone is 1.15

**Pull request merged**